frontend/src/metabase-types/api/settings.ts

```diff
@@ -212,6 +221,7 @@ export interface Settings {
   "slack-files-channel": string | null;
   "slack-token": string | null;
   "slack-token-valid?": boolean;
+  "start-of-week"?: DayOfWeekId;
```

Comment:
I believe it can be either Sunday or Monday, don't you think so? 

<img width="854" alt="image" src="https://user-images.githubusercontent.com/125459446/227210340-0ca04d22-30b8-43d5-868c-4a15c6c032aa.png">

Reply:
my bad, I followed some logic based on datepicker I developed before, so I decided it's possible to have start date only on Sunday or Monday. TIL!

Reply:
actually, I googled only 3 possible options for the start of the week: monday, sunday and saturday. Could the settings in metabase be incorrect?